lib/msf/core/db_manager/import.rb

```diff
@@ -353,6 +355,9 @@ def import_filetype_detect(data)
     elsif (firstline.index("<NessusClientData>"))
       @import_filedata[:type] = "Nessus XML (v1)"
       return :nessus_xml
+    elsif firstline.starts_with?('{"template":')
```

Comment:
I tried to import a Nuclei json result generated with the `-json-export` option and noticed the file starts with `[{"template":` instead of `{"template":`. The import framework is not able to detect the file type.

Am I doing it right?

I'm using Nuclei version 2.9.1.

Reply:
> Am I doing it right?

No. Please re-read the issue description.

If you wanted to describe how the import detection only detects `-o` but not `-json-export` you might have a valid point. but you don't.

Reply:
This PR is intended to handle JSON Lines (JSONL) output. This can be generated with the `-json` flag as per "Testing" section in the PR description.

The `-json-export` option was added to Nuclei last week in https://github.com/projectdiscovery/nuclei/pull/3466. It did not exist when this PR was created. The `-json` flag was renamed to `-jsonl` during introduction of the `-json-export` option.

It makes sense to support both log formats. I'll add it.

Reply:
This PR now supports both JSONL and JSON formats.